package.xml

```diff
@@ -1,31 +1,34 @@
-<package>
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>srdfdom</name>
   <version>0.5.1</version>
   <description>Parser for Semantic Robot Description Format (SRDF).</description>
   <author email="isucan@willowgarage.com">Ioan Sucan</author>
   <author email="gwalck@techfak.uni-bielefeld.de">Guillaume Walck</author>
+  <author email="vmayoral@acutronicrobotics.com">Víctor Mayoral Vilches</author>
+
   <maintainer email="dave@dav.ee">Dave Coleman</maintainer>maintainer>
   <maintainer email="130s@2000.jukuin.keio.ac.jp">Isaac I. Y. Saito</maintainer>maintainer>
   <license>BSD</license>
   <url type="website">http://ros.org/wiki/srdfdom</url>
   <url type="bugtracker">https://github.com/ros-planning/srdfdom/issues</url>
   <url type="repository">https://github.com/ros-planning/srdfdom</url>
-  
-  <buildtool_depend>catkin</buildtool_depend>
 
+  <buildtool_depend>ament_cmake</buildtool_depend>
   <build_depend>boost</build_depend>
   <build_depend>cmake_modules</build_depend>
   <build_depend>libconsole-bridge-dev</build_depend>
-  <build_depend>urdf</build_depend>
+  <build_depend>urdf_model</build_depend>
```

Comment:
This package is not released in melodic or crystal. I believe it should be reverted